security/maltrail/src/etc/rc.d/opnsense-maltrailsensor

```diff
@@ -0,0 +1,52 @@
+#!/bin/sh
+#
+# $FreeBSD$
+#
+# PROVIDE: opnsense-maltrailsensor
+# REQUIRE: SERVERS
+# KEYWORD: shutdown
+#
+
+. /etc/rc.subr
+
+name=maltrailsensor
+
+stop_cmd=maltrailsensor_stop
+start_cmd=maltrailsensor_start
+status_cmd=maltrailsensor_status
+rcvar=maltrailsensor_enable
+pidfile=/var/run/${name}.pid
+command=/usr/sbin/daemon
+command_args="-P /var/run/maltrailsensor.pid python2.7 /usr/local/share/maltrail/sensor.py"
+
+load_rc_config opnsense-maltrailsensor
+
+[ -z "$maltrailsensor_enable" ] && maltrailsensor_enable="NO"
+
+maltrailsensor_status()
```

Comment:
shouldn't status stop and start be automatic ?

Reply:
What do you mean with automatic?

Reply:
boilerplate functions such as status start stop etc should be automatic in the rc framework